fintech-examples/fintech-example-tests/src/test/java/de/adorsys/fintech/tests/e2e/steps/FintechStagesUtils.java

```diff
@@ -9,16 +9,20 @@
 
     public static final String X_REQUEST_ID = "X-REQUEST-ID";
     public static final String PIN = "1234";
-    public static String USERNAME = "bob";
-    public static String FINTECH_LOGIN = "tom";
+    public static String OBA_USERNAME = "htc";
+    public static String OBA_USERNAME_IBAN = "DE59737352520000000108";
     public static final String X_XSRF_TOKEN = "xsrfToken";
     public static final String SESSION_COOKIE = "sessionCookie";
-    public static final String ACCOUNT = "/account/";
+    public static final String ACCOUNT = "/account";
+    public static final String PAYMENT = "/accounts";
```

Comment:
It is not `PAYMENT` it is `TRANSACTIONS` according to the method where it is used - `fintech_can_read_users_accounts_and_transactions`